### PR TITLE
[SMALLFIX] deprecating 'alluxio loadufs' and 'alluxio fs loadMetadata'

### PR DESCRIPTION
**Pull request by jsimsa**

### CODE REVIEW

File: bin/alluxio [hunk 1]
```diff
@@ -254,8 +254,9 @@ function main {
     runJavaClass "$@"
   ;;
   "loadufs")
-    CLASS="alluxio.cli.UfsUtils"
-    runJavaClass "$@"
+    echo "The \"loadufs ...\" command is deprecated since version 1.1."
+    echo "Use the \"fs ls ...\" command instead."
```
Comment (apc999) [inline bin/alluxio]: should we here have `fs ls -R <mountPoint>`?

File: shell/src/main/java/alluxio/shell/command/LoadMetadataCommand.java [hunk 2]
```diff
@@ -51,15 +51,8 @@ protected int getNumOfArgs() {
...
-    AlluxioURI inputPath = new AlluxioURI(args[0]);
-
-    try {
-      LoadMetadataOptions options = LoadMetadataOptions.defaults().setRecursive(true);
-      mFileSystem.loadMetadata(inputPath, options);
-    } catch (AlluxioException e) {
-      throw new IOException(e.getMessage());
-    }
+    System.out.println("The \"loadMetadata\" command is deprecated since version 1.1.");
+    System.out.println("Use the \"ls\" command instead.");
```
Comment (apc999) [inline shell/src/main/java/alluxio/shell/command/LoadMetadataCommand.java]: use fs ls -R command instead?

**Comment by apc999** [comment]
two comments left

File: bin/alluxio (hunk 1 above)
Comment (jsimsa) [inline bin/alluxio]: done

File: shell/src/main/java/alluxio/shell/command/LoadMetadataCommand.java (hunk 2 above)
Comment (jsimsa) [inline shell/src/main/java/alluxio/shell/command/LoadMetadataCommand.java]: loadMetadata is not recursive

**Comment by jsimsa** [comment]
@apc999  thanks for the review, addressed your comments

**Comment by apc999** [comment]
LGTM

**Pull request merged**